src/main/java/millom/sandbox/pojos/Sol.java

```diff
@@ -0,0 +1,188 @@
+package millom.sandbox.pojos;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Sol {
+  @JsonProperty("id")
+  private String id;
```

Comment:
There is no need in `@JsonProperty` when both POJO and JSON field have exactly same name. Just saying :-)

Reply:
Removed @JsonProperty when both POJO and JSON feels have same name.